test/TestProjects/MgmtMockAndSample/tests/Generated/Samples/Sample_DeletedManagedHsmResource_PurgeAManagedHSMPool.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+
+namespace MgmtMockAndSample
+{
+    public partial class Sample_DeletedManagedHsmResource_PurgeAManagedHSMPool
+    {
+        // Purge a managed HSM Pool
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
+        public async Task PurgeDeleted()
+        {
+            // this example is just showing the usage of "ManagedHsms_PurgeDeleted" operation, for the dependent resources, they will have to be created separately.
+
+            // authenticate your client
+            ArmClient client = new ArmClient(new DefaultAzureCredential());
+
+            // this example assumes you already have this DeletedManagedHsmResource created on azure
+            // for more information of creating DeletedManagedHsmResource, please refer to the document of DeletedManagedHsmResource
+            ResourceIdentifier deletedManagedHsmResourceId = MgmtMockAndSample.DeletedManagedHsmResource.CreateResourceIdentifier("00000000-0000-0000-0000-000000000000", "westus", "hsm1");
+            MgmtMockAndSample.DeletedManagedHsmResource deletedManagedHsm = client.GetDeletedManagedHsmResource(deletedManagedHsmResourceId);
+
+            // invoke the operation
+            await deletedManagedHsm.PurgeDeletedAsync(WaitUntil.Completed);
```

Comment:
Can we print some output at the end of the sample?

Reply:
Added